packages/contracts-bedrock/contracts/universal/OptimismMintableERC20Factory.sol

```diff
@@ -41,15 +41,12 @@ contract OptimismMintableERC20Factory {
     /**
      * @notice Address of the StandardBridge on this chain.
      */
-    address public bridge;
+    address public immutable bridge;
 
     /**
-     * @notice Initializer.
-     *
      * @param _bridge Address of the StandardBridge on this chain.
      */
-    function initialize(address _bridge) public {
-        require(bridge == address(0), "OptimismMintableERC20Factory: already initialized");
+    constructor(address _bridge) {
```

Comment:
This is no longer upgradeable, or it doesn't need an initializer because we don't need to change values when changing the impl (ie. the conversation we had the other day).

Reply:
If we want to upgrade this, then we can just deploy a new implementation and then call `upgradeTo` on the implementation since the constructor has no state access